tlovett1/safe-redirect-manager
# Export redirects to csv

### Description of the Change
Added a new WP CLI command to export the redirects into a csv file.

### How to test the Change
To test the feature, please run `wp safe-redirect-manager export_csv` This will create a csv file with the default name srm-redirects.csv. If you want to add a different name for the csv file then you can do something like `wp safe-redirect-manager export_csv --filename=sample-redirects`

*Please note that it is not necessary to provide the .csv extension for the name, it'll be appended automatically.

### Changelog Entry

> Added - WP-CLI command `export` to exporting redirects into a CSV file.

### Credits
Props @zamanq @jeffpaul 


### Checklist:
